WebUI/Data/DataAccess/ReviewService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using ConestogaInsidersClub.Data.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ConestogaInsidersClub.Data.DataAccess
+{
+    public class ReviewService : IReviewService
+    {
+        private readonly ApplicationDbContext context;
+
+        public ReviewService(ApplicationDbContext context)
+        {
+            this.context = context;
+        }
+        public async Task AddReview(Review review)
+        {
+            await context.Reviews.AddAsync(review);
```

Comment:
Correct me if I'm wrong @ANDREYDEN and @MorganDWalker but:

- should we update the schema to make sure that `approved` is nullable? As far as I remember that's what agreed to for the cases when a review has been submitted but has been neither approved nor disapproved by the admin yet.

- and in this case, I think we would need another method for retrieving the reviews, where `Approved == null`.

![image](https://user-images.githubusercontent.com/56138100/140653795-ac3b728c-7af0-44e7-b600-02f482c89774.png)

Reply:
Yes, makes sense, this is as simple as changing `bool` to `bool?` in the `Review` model and creating a migration.